docs/release/fundamentals/queries-and-expressions.ipynb

```diff
@@ -0,0 +1,2023 @@
+       "            </div></td>\n",
+       "      <td>None</td>\n",
+       "      <td>ValueError</td>\n",
+       "      <td>band index out of range</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "                                               image channel  \\\n",
+       "0  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "1  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "2  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "3  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "4  <PIL.JpegImagePlugin.JpegImageFile image mode=...    None   \n",
+       "\n",
+       "  channel_errortype         channel_errormsg  \n",
+       "0        ValueError  band index out of range  \n",
+       "1        ValueError  band index out of range  \n",
+       "2        ValueError  band index out of range  \n",
+       "3        ValueError  band index out of range  \n",
+       "4        ValueError  band index out of range  "
+      ]
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "t.select(t.image, t.channel, t.channel.errortype, t.channel.errormsg).head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6847abcb-1b78-4031-b097-8f4b7263b192",
+   "metadata": {},
+   "source": [
+    "## The Pixeltable Type System"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3dac861b-e4d6-4f91-a55d-13530c3c4e0a",
+   "metadata": {},
+   "source": [
+    "We've seen that every column and every expression in Pixeltable has an associated __Pixeltable type__. In this section, we'll briefly survey the various Pixeltable types and their uses.\n",
+    "\n",
+    "Here are all the supported types and their corresponding Python types:\n",
+    "\n",
+    "| Pixeltable type | Python type |\n",
+    "|-----------------|-------------|\n",
+    "| pxt.String | str |\n",
+    "| pxt.Int | int |\n",
+    "| pxt.Float | float |\n",
+    "| pxt.Bool | bool |\n",
+    "| pxt.Timestamp | datetime.datetime |\n",
+    "| pxt.Json | ** |\n",
+    "| pxt.Array | np.ndarray |\n",
+    "| pxt.Image | PIL.Image.Image |\n",
+    "| pxt.Video | str |\n",
+    "| pxt.Audio | str |\n",
+    "| pxt.Document | str |\n",
+    "\n",
+    "The Python type is what you'll get back if you query an expression of the given Pixeltable type. For `pxt.Json`, it can be any of `str`, `int`, `float`, `bool`, `list`, or `dict`.\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<code>pxt.Audio</code>, <code>pxt.Video</code>, and <code>pxt.Document</code> all correspond to the Python type <code>str</code>. This is because those types are represented by URLs or file paths that reference the media in question.\n",
```

Comment:
You should point out that if you do t.select(t.audio_column) you'll get back the local path, not the url you put in there. The assumption is that you'll want to do something with the actual data.